src/components/DisplayCard/DisplayCard.stories.tsx

```diff
@@ -108,7 +106,11 @@ onlyHeaderWithButton.args = {
   title: "Site settings",
   caption: "Manage site footer, links, logos, and more",
   icon: <Icon as={BiCog} fontSize="1.5rem" />,
-  button: <Button variant="reverse">Manage</Button>,
+  button: (
+    <Text textStyle="subhead-1" color="text.title.brand" marginRight="1rem">
+      Manage
+    </Text>
```

Comment:
being annoying here - but idt using text as a button is ideal. we should change this to be a button and style it accordingly (or use the `link/clear` variant; double check w/ design)

Reply:
Adjusted to Button variant="link" in cd2f64a.